repairnator/src/main/java/fr/inria/spirals/repairnator/Launcher.java

```diff
@@ -243,7 +249,9 @@ private void run(String[] args) throws JSAPException, IOException {
             System.exit(-1);
         }
         this.checkToolsLoaded();
-        this.checkNopolSolverPath();
+        if (RepairMode.valueOf(arguments.getString("mode").toUpperCase()) != RepairMode.FORBEARS) {
```

Comment:
We can initialize the field mode right now and then you can use it in your condition